Source/embeddings/Builder/RemoveDecoratedMethods.ts

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import { Constructor } from '@dolittle/types';
-import { EmbeddingAlreadyHasARemoveDecorator } from './EmbeddingAlreadyHasARemoveDecorator';
-import { EmbeddingClassRemoveMethod } from './EmbeddingClassRemoveMethod';
+import { EmbeddingAlreadyHasADeletionDecorator } from './EmbeddingAlreadyHasADeletionDecorator';
+import { EmbeddingClassDeletionMethod } from './EmbeddingClassDeletionMethod';
 import { RemoveDecoratedMethod } from './RemoveDecoratedMethod';
 
 
 /**
- * Represents the system that knows about all the methods decorated with the @remove() decorator.
+ * Represents the system that knows about all the methods decorated with the @resolveDeletionToEvents() decorator.
  */
 export class RemoveDecoratedMethods {
```

Comment:
```suggestion
export class DeletionDecoratedMethods {
```
We also gotta rename the file (can't make a suggestion for that on GitHub)